angular-ngrx-scss/src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import { environment } from '../environments/environment';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { TokenInterceptor } from './auth/services/token.interceptor';
+import { ClickAwayDirective } from './shared/directives/click-away.directive';
```

Comment:
Getting a lint warning that this is defined but not used - do we need to add this to the imports or declarations in this file too? Or is this one not needed, since it's in shared as well?